slider/curve.py

```diff
@@ -128,13 +123,42 @@ def __call__(self, t):
         return self._curves[bi]((t - pre_t) / (post_t - pre_t))
 
 
-class Passthrough(Curve):
-    kinds = 'P'
+class LinearMetaCurve(MetaCurve):
```

Comment:
Looking at the source of osu! it appears that the Linear curve splits into subcurves between each point: https://github.com/ppy/osu/blob/7fbbe74b65e7e399072c198604e9db09fb729626/osu.Game/Rulesets/Objects/SliderCurve.cs#L30

The particular slider that I saw this was and old favorite: https://osu.ppy.sh/s/19789

For the new lstm stuff I ran through all of my maps and collected stats about all of the locations and the old interpretation here would cause positions on the scale of 1e23.